phpstan-baseline-8+.neon

```diff
@@ -275,6 +275,11 @@ parameters:
 			count: 1
 			path: tests/CsvReadTest.php
 
+		-
```

Comment:
Nebylo jednodušší tam ten `void` dopsat?

Reply:
to bych teď neřešil, protože ten return type není nikde a byl bych za to udělat v novým PR (někdy)